### PR TITLE
8 Profiles Links of Speakers Updated

### LINKED ISSUE #116: 8 Missing Profile Links of Speakers
No Profile Links for Few Speakers

### PR DESCRIPTION
**Pull request by NaviRocker**
fixes #116

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -996,7 +996,7 @@ <h1>Speakers</h1>
 											<a href="https://kushaldas.in"><i class="icon fa fa-external-link"></i></a>
 											<a href="http://github.com/kushaldas"><i class="icon fa fa-github"></i></a>
 											<a href="https://twitter.com/kushaldas"><i class="icon social_twitter"></i></a>
-											<a href="#"><i class="icon social_linkedin"></i></a>
+											<a href="https://www.linkedin.com/company/7900744/"><i class="icon social_linkedin"></i></a>
```
Comment (SaptakS) [inline index.html]: I believe this is the linkedin profile: https://www.linkedin.com/in/kushal-das-5a655b8/

**Review by SaptakS — CHANGES_REQUESTED** [review]

**Review by NaviRocker — COMMENTED** [review]
Thank You

**Comment by NaviRocker** [comment]
Updated

**Comment by akarshan96** [comment]
Looks good to me 👍

**Review by SaptakS — APPROVED** [review]

**Pull request merged**